harness/determined/pytorch/_pytorch_context.py

```diff
@@ -197,12 +209,18 @@ def _init_device(self) -> None:
             self.device = torch.device("cpu")
         check.is_not_none(self.device)
 
-    def _to_device(self, data: pytorch._Data) -> pytorch.TorchData:
+    def to_device(self, data: pytorch._Data) -> pytorch.TorchData:
+        """Map generated data to the device allocated by the Determined cluster.
+
+        All the data in the data loader and the models are automatically moved to the
+        allocated device. This method aims at providing a function for the data generated
+        on the fly.
+        """
         return pytorch.to_device(
             data, self.device, self.warning_logged[_WarningLogs.FAILED_MOVING_TO_DEVICE]
         )
 
-    def _configure_apex_amp(
+    def configure_apex_amp(
```

Comment:
non-blocking: do you think `apex_amp_initialize` is a better name than `configure_apex_amp`?  I kind of like it since the arguments are the same as `apex.amp.initialize()`.

Reply:
I guess `configure_apex_amp` sounds closer to natural language flow. so affirm the original judgment?